src/app/components/Drbb/DrbbContainer.jsx

```diff
@@ -97,10 +106,7 @@ class DrbbContainer extends React.Component {
         key="drbb-link"
       >
         <div className="drbb-promo">
-          <img
-            alt="digital-research-book"
-            src={require('../../../client/assets/drbb_promo.png').default}
-          />
+          { this.promo() }
```

Comment:
curious what was the reasoning behind extracting this?  I generally like markup to be all in one place because I find it easier to read, but that's completely a personal preference.

Reply:
This came out of writing the tests. It did not cause anything to fail, but I got weird logging about loading the png. So, to work around that, I moved it into a function to create a stub.

Reply:
Try asking it to ignore images in your test setup script, like https://github.com/NYPL/nypl-design-system/blob/development/src/__tests__/setupEnzyme.ts#L25